models/movieDao.js

```diff
@@ -0,0 +1,127 @@
+const appDataSource = require('../models/dataSource');
+
+const getMovies = async (main, release, sort) => {
+  try {
+    const sortType = Object.freeze({
+      openingAsc: 'ORDER BY m.opening_date ASC',
+      openingDesc: 'ORDER BY m.opening_date DESC',
+    });
+
+    const rawQuery = `
+    SELECT
+      m.name as movieName,
+      m.english_name as movieNameInEnglish,
+      m.simple_description as movieSimpleDescription,
+      m.thumbnail_image_url as movieThumbnailImageUrl,
+      DATE_FORMAT(m.opening_date, '%Y-%m-%d') as movieOpeningDate,
+      fr.rating as filmAgeRating,
+      keyword.movieKeyword as movieKeyword
+    FROM movies m
+    INNER JOIN film_ratings fr ON fr.id = m.film_rating_id
+    INNER JOIN (
+      SELECT
+        mmk.movie_id,
+        JSON_ARRAYAGG(
+            mk.keyword
+        ) as movieKeyword
+      FROM movies_movie_keywords mmk 
+      INNER JOIN  movie_keywords mk ON mmk.movie_keyword_id = mk.id
+      GROUP BY mmk.movie_id
+    ) keyword ON m.id = keyword.movie_id
+    ${release ? 'WHERE DATE(m.opening_date) < DATE(NOW())' : ''}
+    ${sortType[sort] ? sortType[sort] : 'ORDER BY m.opening_date DESC'}
+    ${main ? 'LIMIT 5' : ''}
+    `;
+    const movies = await appDataSource.query(rawQuery);
+    return movies;
```

Comment:
```suggestion
    return appDataSource.query(rawQuery);
```
변수 선언 없이 곧바로 return 할 수 있겠네요!